primer.md

```diff
@@ -171,17 +171,17 @@ or events on the web, or MQTT and AMQP for connection-oriented messaging and
 telemetry transfers.
 
 Some widely used protocols have become de-facto standards emerging out of strong
-ecosystems of top-level multi-company consortia projects, such as Apache Kafka, 
-and largely in parallel to the evolution of the aforementioned standards stacks.
+ecosystems or top-level multi-company consortia projects and largely in parallel
+to the evolution of the aforementioned standards stacks.
```

Comment:
Can we quantify "multi-company"?

Reply:
as I mentioned in last week's call, given what I now know the intent of this change, I am not even sure this part is necessary since it's a very subject description. We have a more objective definition of de-facto standards in the text below.

Reply:
`multi-company consortia projects` was already accepted, and isn't being changed. If that needs to be defined to be more specific, I could take a stab at that. 

The change here is that it's sufficient to for a project to have a strong ecosystems built, and it's also sufficient to be built by multi-company consortia. Previously the clauses were each necessary and jointly sufficient; now they're each sufficient. I think it's a good change, because there's a lot of widely used technology that didn't emerge from multi-company consortia that we would ideally still interoperate with.

Reply:
I support this change. Just feel it will be better to quantify the "multi-company".  You are right that my comment is not to this PR change, but rather to the existing text. It will be great if this PR can take care of this together. Thanks

Reply:
How about `s/multi-company consortia/consortia of three or more companies`? In context, with some edits to make the grammar work, that becomes:

> Some widely used protocols have become de-facto standards emerging out of strong ecosystems or projects from top-level consortia of three or more companies, created largely in parallel to the evolution of the previously mentioned standards stacks.

Does that work?

Reply:
LGTM

Reply:
@ultrasaurus, does this wording work for you?